markdownlint.js

```diff
@@ -161,7 +161,37 @@ function printResult(lintResult) {
     process.exitCode = 1;
   }
 
-  if (program.output) {
```

Comment:
Can you leave the output section first as it is more common?

Reply:
sure, makes sense